suites/tentacle/nvmeof/tier-3_nvmeof_namespace_limit.yaml

```diff
@@ -0,0 +1,327 @@
+# Test module to explore namespace scale limitations in Single subsystem
```

Comment:
Please remove this suite as well as now we combined scale test suites with GW groups